Tryouts/Prototypes/Shell/Node-launcher/Lib/src/WindowConfig.ts

```diff
@@ -0,0 +1,5 @@
+export interface WindowConfig {
+    url: string;
```

Comment:
Is this config object all or nothing? All these properties seem to be optional (and they are all treated as optional).

Reply:
The parameters are optional because the user might want to set the url, but not the width/height. And vice versa.